ovirt.gemspec

```diff
@@ -36,7 +36,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "coveralls"
 
   spec.add_dependency "more_core_extensions", ">= 3.0.0"
-  spec.add_dependency "nokogiri"
+  spec.add_dependency 'nokogiri', '~> 1.6.8'
```

Comment:
I wish we didn't have to kill support for 1.5.11 even though it seems to be patched at that version.

I think this should be `>= 1.6.8` otherwise this gem will not work with 1.7.0 when it's released.  nokogiri is pretty stable to be "open" 😉